src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalFinalVariableNameCheck.java

```diff
@@ -30,6 +30,11 @@
  *  A catch parameter and resources in try statements
  * are considered to be a local, final variables.
  * </p>
+ * <p>
+ * This check does not support final pattern variables. Instead, use
+ * <a href="patternvariablename.html#PatternVariableName">
```

Comment:
Javadoc has to be fully qualified URL. Only xdoc can't be.

Reply:
Yeah, make sense. done

Reply:
xwiki failure is unrelated

Reply:
@rnveach ping